modules/ROOT/pages/clauses/call-subquery.adoc

```diff
@@ -534,7 +538,9 @@ The amount of work to do in each separate transaction can be specified in terms
 to process before committing the current transaction and starting a new one.
 The number of input rows is set with the modifier `OF n ROWS` (or `ROW`).
 If omitted, the default batch size is `1000` rows.
-The following is the same example but with one transaction every `2` input rows:
+The number of rows can be expressed using any expression that evaluates to a positive integer -- however the expression cannot refer to nodes or relationships.
+
+The following is the same example as above for loading a CSV file but with one transaction every `2` input rows:
```

Comment:
I am not sure whether we should backquote `2` (and also`1000`) a couple lines above. But I leave it up to the documentation team to decide on the style guide.